tsconfig.json

```diff
@@ -1,51 +1,37 @@
 {
```

Comment:
Why did you remove all the comments and entries from here? Please add it back. This is not right according to me.

Reply:
i guess, it was removed by npm, let me do it for you